packages/language-server/src/documentation.ts

```diff
@@ -108,3 +113,30 @@ export function getNodeDetail(node: BaseUI5Node): string {
       return node.name;
   }
 }
+
+export async function getUI5NodeName(
+  offsetAt: number,
+  text: string,
+  model?: UI5SemanticModel,
+  cachePath?: string,
+  framework?: string,
+  ui5Version?: string
+): Promise<BaseUI5Node | undefined> {
+  const documentText = text;
+  const { cst, tokenVector } = parse(documentText);
```

Comment:
Again I do not recall.
Does every flow functional flow re-parses the text each time?
Or is there a cache of documents / CST?